consensus/hotstuff/signature/weighted_signature_aggregator_test.go

```diff
@@ -252,7 +252,7 @@ func TestWeightedSignatureAggregator(t *testing.T) {
 		_, err = aggregator.TrustedAdd(ids[1].NodeID, sigs[1]) // stand-alone verification
 		require.NoError(t, err)
 
-		// Aggregation should validate its own aggregation result and error with sentinel InvalidAggregatedSignatureError
+		// Aggregation should validate its own aggregation result and error with sentinel ErrIdentitySignature
```

Comment:
I think this error is still called `model.InvalidAggregatedSignatureError`, it hasn't been renamed to `ErrIdentitySignature` (yet).

Reply:
changed godoc to `model.InvalidAggregatedSignatureError` in https://github.com/onflow/flow-go/pull/3677/commits/13d72333fe80c5ec9feada0c38d766a6dae1fee0